### PR TITLE
#24 added JSDoc comment block for randomMissionsCompleted

### PR DESCRIPTION
**Pull request by Rob-Rychs**
referenced from this [issue #24](https://github.com/DAVFoundation/missioncontrol/issues/24)

### CODE REVIEW

File: server/simulation/random.js [hunk 1]
```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *   
+ * Returns an object with two properties: one number between 4 and 89, and one number between 2 and 9 
+ * 
+ * @returns {object} with 2 properties,
```
Comment (TalAter) [inline server/simulation/random.js]: Describe the returned object using the same syntax as https://github.com/Rob-Rychs/missioncontrol/blob/db2e4c2891281c4a2837cd9aeac8c909d3686c37/server/simulation/random.js#L57

File: server/simulation/random.js [hunk 2]
```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *
```
Comment (TalAter) [inline server/simulation/random.js]: Remove empty line.

Yes, it is a bit silly. But these comments are later parsed to create API documentation automatically and are also displayed by the IDE, so it is important to keep to a standard.

File: server/simulation/random.js [hunk 3]
```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *   
+ * Returns an object with two properties: one number between 4 and 89, and one number between 2 and 9
```
Comment (TalAter) [inline server/simulation/random.js]: I would perhaps use the text description here (which is for people) to describe the meaning of the function and leave the description of the structure of the returned object (which is for machines) to the `@returns` bit on the bottom.

For example:
````
/**
 * Returns an object with two properties containing a random number of missions completed, and
 * a smaller random number of missions completed in the last 7 days.
````

**Comment by TalAter** [comment]
Hey @Rob-Rychs 

Any questions about the code review? Do you want to modify your pull request, or should I just merge and deal with the issue myself?

**Comment by Rob-Rychs** [comment]
Hey @TalAter 
Thanks a lot for the comments, I will modify my PR.

**Comment by TalAter** [comment]
Fixes #24 

Thank you 💐

**Pull request merged**